e2e/process-services-cloud/process-filter-results.e2e.ts

```diff
@@ -0,0 +1,304 @@
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+        await processInstancesService.suspendProcessInstance(suspendProcessInstance.entry.id, candidateBaseApp);
+
+        completedProcess = await processInstancesService.createProcessInstance(processDefinition.entry.key, candidateBaseApp, {
+            'name': StringUtil.generateRandomString(),
+            'businessKey': StringUtil.generateRandomString()
+        });
+        queryService = new QueryService(apiService);
+
+        const task = await queryService.getProcessInstanceTasks(completedProcess.entry.id, candidateBaseApp);
+        tasksService = new TasksService(apiService);
+        const claimedTask = await tasksService.claimTask(task.list.entries[0].entry.id, candidateBaseApp);
+        await tasksService.completeTask(claimedTask.entry.id, candidateBaseApp);
+
+        await settingsPage.setProviderBpmSso(
+            browser.params.config.bpmHost,
+            browser.params.config.oauth2.host,
+            browser.params.config.identityHost);
+        loginSSOPage.loginSSOIdentityService(testUser.email, testUser.password);
+        await LocalStorageUtil.setConfigField('adf-edit-process-filter', JSON.stringify(editProcessFilterConfigFile));
+        await LocalStorageUtil.setConfigField('adf-cloud-process-list', JSON.stringify(processListCloudConfigFile));
+        done();
+    });
+
+    afterAll(async (done) => {
+        await apiService.login(browser.params.identityAdmin.email, browser.params.identityAdmin.password);
+        await identityService.deleteIdentityUser(testUser.idIdentityService);
+        done();
+    });
+
+    beforeEach(() => {
+        navigationBarPage.navigateToProcessServicesCloudPage();
+        appListCloudComponent.checkApsContainer();
+        appListCloudComponent.goToApp(candidateBaseApp);
+        tasksCloudDemoPage.taskListCloudComponent().checkTaskListIsLoaded();
+        processCloudDemoPage.clickOnProcessFilters();
+    });
+
+    it('[C306887] Should be able to filter by appName', async () => {
+        processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setAppNameDropDown(candidateBaseApp).setProperty('initiator', testUser.email);
+
+        processCloudDemoPage.processListCloudComponent().checkContentIsDisplayedByName(runningProcessInstance.entry.name);
+
+    });
+
+    it('[C306889] Should be able to see "No process found" when using an app with no processes in the appName field', async () => {
+        processCloudDemoPage.editProcessFilterCloudComponent().clickCustomiseFilterHeader().setAppNameDropDown('subprocessapp').setProperty('initiator', testUser.email);
+
+        expect(processListPage.checkProcessListTitleIsDisplayed()).toEqual('No Processes Found');
+
+    });
+
+    it('[C306890] Should be able to filter by initiator', async () => {
```

Comment:
I would have added one more check here related to the fact that a process instance started by a different user is not displayed.